astropy/timeseries/tests/test_binned.py

```diff
@@ -89,6 +89,19 @@ def test_initialization_time_bin_start_scalar():
     assert exc.value.args[0] == ("'time_bin_start' is scalar, so 'time_bin_size' is required")
 
 
+def test_initialization_n_bins_invalid_arguments():
+
+    # Make sure things crash when when n_bins is passed as an argument while
```

Comment:
"things crash" is commonly understood as the interpreter raising a segmentation fault or other more "severe" issues. Maybe change this as:

```suggestion
    # Make sure an exception is raised when n_bins is passed as an argument while
```
P.S. Please don't be overwhelmed by the number of nitpicky comments. This PR is basically ready, but making sure that the code looks "good" is important.